spec/requests/tool_integration/baby_cost_calculator_spec.rb

```diff
@@ -1,14 +1,29 @@
-RSpec.describe ToolMountPoint::BabyCostCalculator, type: :request do 
+RSpec.describe ToolMountPoint::BabyCostCalculator, type: :request do
   %W(
-    /en/tools/#{ToolMountPoint::BabyCostCalculator::EN_ID}
-    /cy/tools/#{ToolMountPoint::BabyCostCalculator::CY_ID}
-  ).each do |path|
-    describe path do
-      before do
-        get path
+    en
+    cy
+  ).each do |locale|
+    context "locale #{locale}" do
+      let (:path) { "/#{locale}/tools/" + "ToolMountPoint::BabyCostCalculator::#{locale.upcase}_ID".constantize }
+
+      describe 'non-syndicated tool request' do
+        before do
+          get path, nil, { 'X-Syndicated-Tool' => false }
+        end
+
+        it "redirects to article baby cost calculator" do
+          expect(response.status).to eq 301
+          expect(response).to redirect_to article_path('baby-costs-calculator', locale: locale)
```

Comment:
Hey @habitullence this redirect will work but, with a hidden gotcha :)

Whereas, https://www.moneyadviceservice.org.uk/en/articles/baby-costs-calculator will work. The Welsh link https://www.moneyadviceservice.org.uk/cy/articles/baby-costs-calculator will not work. 

The correct welsh link is https://www.moneyadviceservice.org.uk/cy/articles/cyfrifiannell-costau-babi